# part location and part placement target location

string initial_preset_location
string final_preset_location
geometry_msgs/Pose part_pose
geometry_msgs/Pose part_target_pose
float64 part_offset

---
bool is_assembly_done